handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -889,7 +889,8 @@ public final SSLEngineResult wrap(
                             // to write encrypted data to. This is an OVERFLOW condition.
                             // [1] https://www.openssl.org/docs/manmaster/ssl/SSL_write.html
                             return newResult(BUFFER_OVERFLOW, status, bytesConsumed, bytesProduced);
-                        } else if (sslError == SSL.SSL_ERROR_WANT_X509_LOOKUP) {
+                        } else if (sslError == SSL.SSL_ERROR_WANT_X509_LOOKUP
+                                || sslError == SSL.SSL_ERROR_WANT_PRIVATE_KEY_OPERATION) {
```

Comment:
nit: I see you are mixing two styles:

```java
if (a ||
    b) { ... }

if (a
    || b) { ... }
```

IIRC we preferred the former since the beginning of the project. ;-)